Model/Import/AbstractRatesConsumer.php

```diff
@@ -117,6 +119,8 @@ public function process(\Magento\AsynchronousOperations\Api\Data\OperationInterf
         $this->reset();
 
         try {
+            $this->validate();
+            $this->setData();
```

Comment:
The catch block for a generic exception below (line 130) needs updated to reference our TaxJar process rather then the product attributes update.